packages/search/lib/commands/ALIASADD.ts

```diff
@@ -1,3 +1,5 @@
+export const FIRST_KEY_INDEX = 1;
```

Comment:
should be `2`?

Reply:
both alias and index need to be same slot, else

```
127.0.0.1:6379> FT.ALIASADD test:{node-2} idx:{node-1}
(error) CROSSSLOT Keys in request don't hash to the same slot
```

changed FIRST_KEY_INDEX to 2 as seems more reasonable 👍